pkg/lib/tmp/copy.go

```diff
@@ -0,0 +1,54 @@
+package tmp
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+// CopyTmpDB reads the file at the given path and copies it to a tmp directory, returning the copied file path or an err
+func CopyTmpDB(original string) (path string, err error) {
+	dst, err := ioutil.TempFile("", "db-")
```

Comment:
Note: there's a separate bug that corrects for filesystems w/o a `/tmp` directory.